scripts/null_route_helper

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
+
+"""
+Utility for blocking and unblocking traffic from given source ip address on ACL tables.
+
+The block operation will insert a DENY rule at the top of the table. The unblock operation
+will remove an existing DENY rule that has been created by the block operation (i.e. it does
+NOT insert an ALLOW rule, only removes DENY rules).
+
+Since SONiC supports multi ACL rules share the same priority, all ACL rules created by null_route_helper will
+use the highest priority(9999).
+
+Example:
+
+Block traffic from 10.2.3.4/32:
+./null_route_helper block acl_table_name 10.2.3.4/32
```

Comment:
>/32 [](http://example.com/codeflow?start=49&length=3)

we can remove the prefix completely from CLI

Reply:
Thanks, I will update.